### DISCUSSION TITLE
Can a man be at all good? Or power corrupts the human being.

### DISCUSSION THREAD

**Discussion by kelwinmarcelo** [Schweinepriester/github-profile-achievements#379] [Q&A] [answered]
There is a question that i have been looking for a answer.
It can be? I guess no!

**Comment by PatrickDSiqueira** [comment, marked as answer]
Man is always led to corruption, but it is his duty to continue in the natural essence of goodness

**Reply by kelwinmarcelo** [reply]
I accept your comentary!
